symplyphysics/laws/thermodynamics/average_speed_in_maxwell_boltzmann_statistics.py

```diff
@@ -12,34 +27,52 @@
 from symplyphysics.core.expr_comparisons import expr_equals
 from symplyphysics.laws.thermodynamics.maxwell_boltzmann_statistics import speed_distribution
 
-# Description
-## The average (mean) speed is the expected value of the speed distribution.
+average_speed = Symbol("average_speed", units.velocity, positive=True)
+r"""
+Average molecular speed.
 
-# Law: <v> = sqrt((8 / pi) * (k * T / m))
-## <v> - average molecular speed
-## k - Boltzmann constant
-## T - equilibrium temperature
-## m - molecular mass
+Symbol:
+    :code:`avg(v)`
 
-# Conditions
-## - Assuming the particles are distributed according to Maxwell-Boltzmann statistics.
+Latex:
+    :math:`\langle v \rangle`
```

Comment:
Why do we use angular brackets here and bar in another place? It seems they are different averages but it is quite hard to understand the difference. We should clearly distinguish different averages in symbol names as well as in documentation.

Reply:
They are the same averages, I just wasn't sure which ones were better. I guess the angle ones are more universally applicable and more readable.

Reply:
Yes, let's use angular brackets. And make sure they are the same average. Because there are many of them.

Reply:
For now, there's time average and ensemble average and for the most part they are the same (unless the system is [non-ergodic](https://en.wikipedia.org/wiki/Ergodic_hypothesis)).